### PR TITLE
Adding the command to run tailscaled on start

### PR DESCRIPTION
**Pull request by myusuf3**
working to see if this is something that could be supported with docker within unraid. 

need to figure out how to persist keys between launches.

---
This change is [Reviewable](https://reviewable.io/reviews/tailscale/tailscale/236)

### CODE REVIEW

**Comment by danderson** [comment]
Looks good. Can you address the DCO bot complaint? (you can add a signoff to your commit with `git commit --amend --signoff` if you agree with the DCO terms, then push -f)

Also remove the WIP if this is ready to merge? Looks fine to me.

**Comment by myusuf3** [comment]
Signed off! @danderson you figure out what that issue with tailscaled up what was?

**Comment by myusuf3** [comment]
Any word on this?

File: Dockerfile [hunk 1]
```diff
@@ -17,3 +17,8 @@ RUN go install -v ./cmd/...
 FROM alpine:3.11
 RUN apk add --no-cache ca-certificates iptables
 COPY --from=build-env /go/bin/* /usr/local/bin/
+
+CMD ["/usr/local/bin/tailscaled", "--state=/var/lib/tailscale/tailscaled.state", "--socket=/var/run/tailscale/tailscaled.sock", "--port=41641"]
+
+
+# docker run --network=host  --cap-add=NET_ADMIN --device /dev/net/tun:/dev/net/tun myusuf3/tailscale
```
Comment (bradfitz) [inline Dockerfile:24]: We don't want `myusuf3` in there.

Also, we already added docs to this file earlier:

https://github.com/tailscale/tailscale/blob/master/Dockerfile#L5

The --state and --socket flags weren't required for me to make it work.

**Comment by myusuf3** [comment]
ok @bradfitz

**Pull request closed without merging**